drivers/media/i2c/imx296.c

```diff
@@ -591,7 +591,7 @@ static int imx296_setup(struct imx296 *sensor)
 	imx296_write(sensor, IMX296_CTRL418C, sensor->clk_params->ctrl418c,
 		     &ret);
 
-	imx296_write(sensor, IMX296_GAINDLY, IMX296_GAINDLY_NONE, &ret);
+	imx296_write(sensor, IMX296_GAINDLY, IMX296_GAINDLY_1FRAME, &ret);
```

Comment:
Commit text:
>In order to behave more simlar to the rolling shutter sensors, set the gain dealy to 1 frame. This simplifies userland control of the gain value.

"In order to behave in a similar manner to rolling shutter sensors..."
s/dealy/delay

Reply:
Commit text still reads oddly, and has a typo in ```simlar```

Reply:
Arghhhhh again!